services/tasks.py

```diff
@@ -190,10 +231,21 @@ def run(self):
         """
         Queues all services to be polled for metrics. Should be run via beat.
         """
+
         services = Service.objects.all()
         for service in services:
             service_metric_sync.apply_async(
                 kwargs={"service_id": str(service.id)})
+
+            key = "services.downtime.%s.sum" % (
+                utils.normalise_string(service.name))
+            check = WidgetData.objects.filter(service=None, key=key)
+            if check.count() == 0:
```

Comment:
`.exists()` is possibly faster for the database to do here, since it only checks if it can find each one, it doesn't have to iterate through all of them counting them.